pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -518,11 +506,16 @@ func TestGetPipelineFunc_VerifyNoError(t *testing.T) {
 	tektonclient := fake.NewSimpleClientset()
 
 	unsignedPipeline := test.GetUnsignedPipeline("test-pipeline")
+	unsignedV1Pipeline := &v1.Pipeline{}
+	unsignedPipeline.ConvertTo(ctx, unsignedV1Pipeline)
+	unsignedV1Pipeline.APIVersion = "tekton.dev/v1"
+	unsignedV1Pipeline.Kind = "Pipeline"
+
```

Comment:
we already have tests for fetching a v1 remote pipeline (the ones Yongxuan added in https://github.com/tektoncd/pipeline/pull/6765), e.g. TestGetPipelineFunc_V1Pipeline_VerifyNoError. It looks like this updates tests to fetch a v1 pipeline instead of a v1beta1 pipeline, but I think we want to maintain coverage for fetching v1beta1 pipelines here. (This test class should probably be named  TestGetPipelineFunc_v1beta1Pipeline_VerifyNoError.)

Reply:
Makes perfect sense. This part was outdated as it was introduced before the v1 Trusted Resources verifications PR related with PipelineRef/ TaskRef gets in. 

I opened https://github.com/tektoncd/pipeline/pull/6803 for the updates since it could be a separate refactor from this swap if that helps.